lessons/Algebra2/exploring-linear-models/langs/en-us/pages/other-forms-linear-models.adoc

```diff
@@ -0,0 +1,77 @@
+= Other Forms of Linear Models
+++++
+<style>
+.studentAnswerMedium { min-width: 8em !important; }
+body.workbookpage .fitbruby { padding-top: 1.1rem; }
+body:not(.LessonPlan, .narrativepage) p { line-height: 1.2; }
+.center { padding: 0; }
+</style>
+++++
+
+For this page, you'll need to the @starter-file{alg2-states} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@n Evaluate @show{(code '(lr-plot states-table "state" "college-or-higher" "median-income"))}. What is @math{R^2}? @fitb{5em}{@ifsoln{}}
+
+@n Instead of using function notation, write the **Slope-Intercept form** of the optimal linear model below.
+
+@center{
+ @math{y =} @fitbruby{8em}{@ifsoln{@math{1142}}}{slope}x + @fitbruby{8em}{@ifsoln{20868.14}}{y-intercept}
+}
+
+@vspace{2ex}
+
+**Linear models can be expressed in several forms. In addition to Slope Intercept form, there are also "Standard" and "Point-Slope" forms. Depending on what you want to _do_ with a model, it can be more convenient to use one form instead of another!**
+
+@vspace{2ex}
+
+[cols="^1a,^1a,^1a", stripes="none", options="header"]
+|===
+| Slope-Intercept Form
+| General Linear ("Standard") Form
+| Point-Slope Form
+
+| @math{ y = mx + b}
+| @math{ Ax + By = C}
+| @math{ y-y_1 = m(x-x_1)}
+|===
+
+@vspace{2ex}
+
+@n Write the General Linear (or "Standard") form of the **optimal** linear model below.
+
+@center{
+  @fitbruby{10em}{@ifsoln{1142}}{A}@math{x + } @fitbruby{10em}{@ifsoln{1}}{B} @math{ = } @fitbruby{10em}{@ifsoln{-20864.14}}{C}
+}
+
+@vspace{2ex}
+
+@n Write the Point-Slope form of the **optimal** linear model below.
+
+@center{
+  @math{y - }@fitbruby{10em}{@ifsoln{20868.14}}{y1} @math{ = m(x - } @fitbruby{10em}{@ifsoln{0}}{x1})
+}
+
+@vspace{2ex}
+
+@n Choose one **other** linear model you came up with, and write the General Linear (or "Standard") form of that model below.
```

Comment:
If we're having them choose another model, we should have them tell us what it is and record the slope-intercept form here.

Reply:
Done.